admin/src/templates/forms_json.html

```diff
@@ -19,12 +19,14 @@
   <fieldset>
     <legend translate>Installed Forms</legend>
     <ul>
-      <li ng-repeat="form in forms" class="row">
-        <div class="col-sm-1" ng-bind-html="form.meta.icon | resourceIcon"></div>
-        <div class="col-sm-1">{{form.meta.code}}</div>
-        <div class="col-sm-10" ng-if="form.meta.translation_key" translate>{{form.meta.translation_key}}</div>
-        <div class="col-sm-10" ng-if="!form.meta.translation_key">{{form.meta.label | translateFrom}}</div>
-      </li>
+      <div class=" container-fluid">
```

Comment:
It's not legal to put a `div` as a child of a `ul`. Can you put the `div` as the parent instead?